src/Admin/Model/UserAdmin.php

```diff
@@ -20,13 +20,26 @@
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Show\ShowMapper;
 use Sonata\UserBundle\Form\Type\SecurityRolesType;
+use Sonata\UserBundle\Model\UserManagerInterface;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 /**
  * @phpstan-extends AbstractAdmin<\Sonata\UserBundle\Model\UserInterface>
  */
 class UserAdmin extends AbstractAdmin
 {
+    protected UserManagerInterface $userManager;
+
+    public function __construct(UserManagerInterface $userManager)
+    {
+        $this->userManager = $userManager;
```

Comment:
```suggestion
        parent::__construct();
        
        $this->userManager = $userManager;
```

to please phpstan

Reply:
Added, but not sure if it is better to add to the baseline.

Reply:
I think it's ok to have this "useless" call so far. It will be removed for the next sonata admin major.